demo/api/src/common/blocks/product-list.block.ts

```diff
@@ -0,0 +1,26 @@
+import { BlockData, BlockField, BlockInput, blockInputToData, createBlock } from "@comet/cms-api";
+import { IsArray, IsEnum } from "class-validator";
+
+export enum ProductList {
+    cap = "cap",
+    shirt = "shirt",
+    tie = "tie",
+}
```

Comment:
Could we reuse the enum from here? https://github.com/vivid-planet/comet/blob/main/demo/api/src/products/entities/product-type.enum.ts